tasks/main.yml

```diff
@@ -86,14 +86,25 @@
   template: src='pxe_nodes.json.j2' dest='/var/www/provision/nodes/pxe_nodes.json' backup=yes
   tags: pxe_data
 
-- name: create_kickstart_group_files
+- name: create_kickstart_group_files for legacy nodes
   template: src=kickstart.j2 dest="/var/www/html/kickstart/{{item}}.ks" validate='/usr/bin/ksvalidator %s' backup=yes
   tags: dhcp_kickstart_config
   with_items: "{{ groups|sort }}"
   when:
     - item not in dhcp_kickstart_skip_these_groups
     - groups[item][0] is defined
     - hostvars[groups[item][0]]['os_disks'] is defined
+    - hostvars[groups[item][0]]['target_os_centos_8stream'] is undefined
```

Comment:
Mieluummin kuin `target_os_centos_8stream` tämän voisi rakentaa sellaiseksi `target_os == "centos_8_stream"`

Reply:
Vanhan kickstartin ajamiseksi tarkistettaisiin toki onko tuon muuttujan arvo `centos_7`

Reply:
Näin tehtynä tässä on sekin ongelma, että tuohon muuttujaan on tallennettuna boolean True tai False - jos muuttujalle on annettu mikä tahansa arvo, se on defined, eli tässä tapauksessa `target_os_centos_8stream:  True` ja `target_os_centos_8stream: False` molemmat asentaisi CentOS 8 Streamin; vain jos tuota muuttujaa ei ole annettu konffissa lainkaan asennetaan CentOS 7.

Reply:
Nyt korjattu siten, että tuo legacy kickstart template ajetaan silloin, kun target_os ei ole määritelty.